### PR TITLE
Cleanup ci-setup.sh

### PR DESCRIPTION
**Pull request by selaux**

### CODE REVIEW

File: .ci/ci-setup.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+env
+which rustc
+rustc -V
+which cargo
+cargo -V
+cargo clippy -- -V
+cargo fmt -- -V
+which cmake
+cmake --version
+command -v sccache && sccache -V || echo "sccache not available"
```
Comment (ja2-derek) [inline .ci/ci-setup.sh:78]: Now that we have `sccache` on all platforms, we probably don't need the existence check.

Comment (selaux) [reply .ci/ci-setup.sh]: :+1: Removed.

**Comment by selaux** [comment]
Giving this up, I dont have the patience for this anymore....

**Comment by ja2-derek** [comment]
I like your idea here. Mind if I try to continue from here?

**Comment by selaux** [comment]
Nope, go for it

**Comment by ja2-derek** [comment]
This is done now.

**Pull request merged**